src/frontend/OverlayManager/ExtensionManager/index.tsx

```diff
@@ -119,6 +119,7 @@ const BrowserExtensionManager = function ({
       {showOverlay ? (
         <>
           <div className={BrowserExtensionManagerStyles.bgFilter}></div>
+          <div className={`${BrowserExtensionManagerStyles.closeOverlayText} title`}>{t('hyperplayOverlay.closeOverlay', 'Press Alt or Option + X to close the overlay')}</div>
```

Comment:
Could we split the message based on the platform?
So if it is macOS we show the OPT + X and for the rest we show ALT + X?

Reply:
Agree, I was confused the first time I used it.

Reply:
This could be confusing because you can use keyboards with an alt key on mac and then the keybind from the player's view is alt+X, not option + X, on mac.

Reply:
updated the keybinds to be platform dependent. they use the same i18n strings but with markup variables